### PR TITLE
Teams

### PR DESCRIPTION
**Pull request by MustafaTRKYLMZ**
teams table added to strapi

### CODE REVIEW

File: packages/types/team.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { StrapiBase } from './strapi'
+
+export type TeamBase = StrapiBase & {
+  name: string
+  description: string
+}
+
+export type TeamRelation = {
+  members: Profile[]
+  platforms: Platform
```
Comment (7alip) [inline packages/types/team.ts:12]:
```suggestion
  platform: Platform
```

Comment (7alip) [reply packages/types/team.ts:12]: You could add PlatformRelationInput and other mutation related types.

File: packages/types/strapi.ts [hunk 2]
```diff
@@ -169,6 +170,7 @@ export type StrapiModel =
   | RecommendedTopic
   | RecommendedTweet
   | Subscriber
+  | Team
```
Comment (7alip) [inline packages/types/strapi.ts:173]: TeamCreateInput and TeamUpdate input types could be added.

**Review by 7alip — CHANGES_REQUESTED** [review]
I merged dev to fix conflicts on generated files

**Comment by MustafaTRKYLMZ** [comment]
What should we do know? Can you look for feedback  ? @7alip

File: packages/ui/components/AddMemberDialog/AddMemberDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+              id="member-select"
+              options={options}
+              placeholder="Select members"
+              multiple
+              onChange={e => handleMemberSelect(e as SelectOption[])}
+              value={options.filter(option =>
+                selectedMembersId.includes(option.value),
+              )}
+              {...rest}
+            /> */}
```
Comment (7alip) [inline packages/ui/components/AddMemberDialog/AddMemberDialog.tsx:136]: This should be fixed @MustafaTRKYLMZ

**Pull request merged**